backend/yandex/yandex_crawl.py

```diff
@@ -0,0 +1,49 @@
+import requests
+from bs4 import BeautifulSoup
+
+
+def to_dict(index, sauce, img_sauce):
+    table = {
+         "img_link": img_sauce,
+         "source": sauce,
+         "host": "Yandex",
+         "tittle": "yandex"+str(index)
+         }
+    return table
+
+
+def spider(max_pic_num):
```

Comment:
Please rename function to sth that is easy for other people, like `yandex_crawl` or sth.

Please give it a docstring.